tests/unit/test_package_managers.py

```diff
@@ -42,13 +42,16 @@
 CONTAINER_CENTOS_7 = Container(url='quay.io/centos/centos:7')
 CONTAINER_UBUNTU_2204 = Container(url='docker.io/library/ubuntu:22.04')
 CONTAINER_FEDORA_COREOS = Container(url='quay.io/fedora/fedora-coreos:stable')
+# Local image created via `make image-alpine`, reference to local registry
+CONTAINER_ALPINE = Container(url='containers-storage:localhost/alpine:tmt-unit-tests')
```

Comment:
Isn't this breaking the alpine tests? `containers-storage:localhost/alpine:tmt-unit-tests` - https://dcermak.github.io/pytest_container/usage.html#testing-local-images says `local = Container(url="containers-storage:my/local/image/name")`, i.e. no `localhost`. `containers-storage:alpine:tmt-unit-tests` could work.

Reply:
no, they were broken by missing image building in the unit tests

Reply:
```
❯ podman images
REPOSITORY                                                TAG                  IMAGE ID      CREATED        SIZE
localhost/alpine                                          tmt-unit-tests       47eda9a1ab47  7 hours ago    9.79 MB
```

Maybe also alpine would work, but yeah, maybe a moot point

Reply:
@happz TBC, this was missing:

https://github.com/teemtee/tmt/pull/2788/files#diff-c095f6087b1c0241879c69fef60d24224aa2bf9fe6ec55c18af66d1cfbb2a92eR39